examples/darwin-framework-tool/main.mm

```diff
@@ -30,13 +30,15 @@
 
 int main(int argc, const char * argv[])
 {
-    Commands commands;
-    registerCommandsPairing(commands);
-    registerCommandsInteractive(commands);
-    registerCommandsPayload(commands);
-    registerClusterOtaSoftwareUpdateProviderInteractive(commands);
-    registerCommandsStorage(commands);
-    registerCommandsTests(commands);
-    registerClusters(commands);
-    return commands.Run(argc, (char **) argv);
+    @autoreleasepool {
+        Commands commands;
+        registerCommandsPairing(commands);
+        registerCommandsInteractive(commands);
+        registerCommandsPayload(commands);
+        registerClusterOtaSoftwareUpdateProviderInteractive(commands);
+        registerCommandsStorage(commands);
+        registerCommandsTests(commands);
+        registerClusters(commands);
+        return commands.Run(argc, (char **) argv);
+    }
```

Comment:
Yes, in https://github.com/project-chip/connectedhomeip/pull/22606.  It's needed to avoid leaks in both PRs, from different API changes.  I wanted to get the CI running in parallel and they should merge cleanly.